### PR TITLE
ImageCaption multiline issue in Firefox 

### LINKED ISSUE #834: Issue with Image caption, Multiline support in Firefox
it's a issue when using the plugin,

https://github.com/orthes/medium-editor-insert-plugin/issues/230

### PR DESCRIPTION
**Pull request by ryandh**
https://github.com/orthes/medium-editor-insert-plugin/issues/230

Fix #834

### CODE REVIEW

File: src/js/core.js [hunk 1]
```diff
@@ -141,7 +141,9 @@
                 this.options.ownerDocument.execCommand('unlink', false, null);
             } else if (!event.shiftKey && !event.ctrlKey) {
                 // only format block if this is not a header tag
-                if (!/h\d/.test(tagName)) {
+                // calling formatBlock on firefox will cause incoorect layout if pressed enter in the figcaption
+                // #https://github.com/orthes/medium-editor-insert-plugin/issues/230
+                if (!/h\d|figCaption/i.test(tagName)) {
```
Comment (j0k3r) [inline src/js/core.js]: Shouldn't we use `MediumEditor.utils.blockContainerElementNames` here ?

Comment (ryandh) [inline src/js/core.js]: good point, I think using MediumEditor.utils.blockContainerElementNames makes more sense.

**Comment by nmielnik** [comment]
In addition to using `blockContainerElementNames`, would you be able to add/update any tests around this?

**Comment by nmielnik** [comment]
This work is completed in #986

**Pull request closed without merging**